remove Transifex configuration (#284)
PIH/openmrs-config-pihemr: README.md

Message properties files are used to localize the PIH EMR into different languages.  Currently we support French,
Haitian Kreyol, and Spanish, and there's a separate messages_<lang>.properties files for each within the
configuration/messagesproperties folder.

> To add/update translations, these files can be editing directly (we no longer use Transifex for message codes in
> this repo).

Reviewer: ```suggestion
To add or update translations, edit these files directly. We no longer use Transifex for message codes in
this repo.```
